plugins/source/azure/scripts/update_azure_subpackages.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+set -e
```

Comment:
It's better to have this as bash given we invoke number of subcommands (rather then having just number of `os.Exec`), also it's a pre step to `codegen0`.

go pkg is right source of truth to find what go package are available imo.

Reply:
I agree we shouldn't do this using `os.Exec` but we can HTTP request the file using Go code instead of `curl` and also parse it via Go code regex. This way the logic that does discovery will be in the same place

Reply:
we need to call `go get` which will be weird in imo calling it from a `go` program.

Reply:
So maybe we can have `codegen0` call that script? I'm trying to think of ways to make it easier to discover the different parts of the generate logic. I know it's in `CONTRIBUTING.md`, but we should try making it easier to follow the different parts

Reply:
When I use it locally and debug I don't call the `scripts/update_all_modules` everytime. 1) because it's slow. 2) I want to update my `go.mod` once and then iterate from their with static `go.mod` that doesn't change everytime I run it. I think it's an over optimisation of `./scripts/update_modules && make codegen` or `make codegen` especially given the only once that will be running it will be CI and myself as this will be rarely changed/touched.